lib/src/mapbox_map.dart

```diff
@@ -28,7 +28,7 @@ class MaplibreMap extends StatefulWidget {
     this.trackCameraPosition = false,
     this.myLocationEnabled = false,
     this.myLocationTrackingMode = MyLocationTrackingMode.None,
-    this.myLocationRenderMode = MyLocationRenderMode.COMPASS,
+    this.myLocationRenderMode = MyLocationRenderMode.NORMAL,
```

Comment:
I think this is a good approach, but we should note this in the release notes, since it's a breaking change.

Reply:
Thanks for writing the changelog for me :)

Reply:
Should I adjust the documentation to highlight that you only need location permission if you use specific properties?